src/en/hentaidex/src/eu/kanade/tachiyomi/extension/en/hentaidex/HentaiDex.kt

```diff
@@ -0,0 +1,198 @@
+package eu.kanade.tachiyomi.extension.en.hentaidex
+
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.network.interceptor.rateLimit
+import eu.kanade.tachiyomi.source.model.Filter
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.ParsedHttpSource
+import eu.kanade.tachiyomi.util.asJsoup
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.Request
+import okhttp3.Response
+import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
+import java.text.SimpleDateFormat
+import java.util.Locale
+import kotlin.concurrent.thread
+
+class HentaiDex : ParsedHttpSource() {
+
+    override val name = "HentaiDex"
```

Comment:
website calls itself DEXHentai

Reply:
It seems like the website also identifies as HentaiDex 
![image](https://github.com/user-attachments/assets/80d612fa-d6ee-403e-9417-c108bacfb57e)